### DISCUSSION TITLE
How to get user real ip?

### DISCUSSION THREAD

**Discussion by Edemilorhea** [Chevereto/Chevereto-Free#175] [General]
My chevereto is stock on docker, it is well but i find out  every user login, upload ip  is my docker ip.

maybe I should be setting proxy?

does anyone know?

**Comment by rodber** [comment]
If you don't get the real IP is because your server is under a proxy. This is covered in the docs: https://chevereto-free.github.io/setup/server/requirements.html#real-connecting-ip
